docs/index.md

```diff
@@ -178,31 +178,32 @@ perform further structure manipulation or analyses.
 Here are some quick examples of the core capabilities and objects:
 
 ```python
-import pymatgen.core as pmg
+from pymatgen.core import Element, Composition, Lattice, Structure, Molecule
```

Comment:
Ah my bad, I manually typed in these import yesterday, forgot to sort them, should be:
```python
from pymatgen.core import Composition, Element, Lattice, Molecule, Structure
```

Would fix this in a following PR.